#goal definition
bool reqSDKControl
---
#result definition
bool accept
---
#feedback
bool isSDKControl
